lighthouse-cli/test/smokehouse/test-definitions/perf/expectations.js

```diff
@@ -176,15 +176,15 @@ module.exports = [
           },
         },
         // TODO: uncomment when Chrome m84 lands
```

Comment:
Will delete this TODO once I verify the smoke test can pass in the CI runs.

Reply:
#10976 will take care of this.

Reply:
can you rm this change